proto-public/pbmesh/v2beta1/upstreams_configuration.proto

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) HashiCorp, Inc.
-// SPDX-License-Identifier: MPL-2.0
```

Comment:
I feel like these should still be MPL-2.0, like the `api`. Especially since they make their way into `consul-k8s`. Same comment for multiple files.

Reply:
yep, you're right! great catch